sonaranalyzer-dotnet/tests/SonarAnalyzer.UnitTest/TestCases/EmptyCollectionsShouldNotBeEnumerated.cs

```diff
@@ -723,4 +723,52 @@ public Program(string id)
             if (string.IsNullOrEmpty(id)) throw new ArgumentNullException(nameof(id));
         }
     }
+
+    // Large CFG that causes the exploded graph to hit the exploration limit
+    // See https://github.com/SonarSource/sonar-csharp/issues/767 (comments!)
```

Comment:
Probably better to link directly the comment (https://github.com/SonarSource/sonar-csharp/issues/767#issuecomment-362338301)